core/controllers/reader.py

```diff
@@ -346,7 +410,12 @@ class ReaderFeedbackHandler(base.BaseHandler):
 
     @require_playable
     def post(self, exploration_id):
-        """Handles POST requests."""
+        """Handles POST requests.
+        Records a thread_id of the feedback from a learner.
```

Comment:
Move to end of previous line, and ensure there's a newline between this and the "Args:" line. DItto elsewhere.

Also, change to: Records the feedback submitted by a learner to an exploration.